locales/es_ES.json

```diff
@@ -0,0 +1,107 @@
+{
```

Comment:
File should simply be named "es" according to lang code standards found [here](https://andiamo.co.uk/resources/iso-language-codes).

Reply:
Assumed the other notation was the one to use because that's what was printed to the console, changed (and the rest of the review).